app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -44,10 +44,19 @@ const ListingDetail = () => {
             <ListingDetailsImageCard listing={listing} />
             <ListingDetailsPricingTable listing={listing} />
             <ListingDetails>
-              <ListingDetailsEligibility listing={listing} />
-              <ListingDetailsFeatures listing={listing} />
-              <ListingDetailsNeighborhood />
-              <ListingDetailsAdditionalInformation listing={listing} />
+              <ListingDetailsEligibility
+                listing={listing}
+                imageSrc={getAssetPath("listing-eligibility.svg")}
+              />
+              <ListingDetailsFeatures
+                listing={listing}
+                imageSrc={getAssetPath("listing-features.svg")}
+              />
+              <ListingDetailsNeighborhood imageSrc={getAssetPath("listing-neighborhood.svg")} />
+              <ListingDetailsAdditionalInformation
+                listing={listing}
+                imageSrc={getAssetPath("listing-legal.svg")}
+              />
```

Comment:
Do we think the images may change with the usage of the components? If not, we may want to just put the image info in the components themselves.

Reply:
I wasn't sure since the module components seem pretty specific to a set of content atm, and adding the context for `getAssetPath` on each component feels a bit heavy